dspace-spring-rest/src/test/java/org/dspace/app/rest/matcher/SubmissionFormFieldMatcher.java

```diff
@@ -87,4 +89,38 @@ private SubmissionFormFieldMatcher() { }
             hasJsonPath("$.hints", containsString(hints))
         );
     }
+
+    public static Matcher<? super Object> matchFormOpenRelationshipFieldDefinition(String type, String label,
```

Comment:
Please add Javadocs, just like other matchers.  I think this is especially important here because the phrase "open relationship" isn't defined anywhere in this code... so this method name isn't self-explanatory

Reply:
The JavaDocs have been included